docs/spfx/sharepoint-2019-support.md

```diff
@@ -23,35 +24,171 @@ Starting from v1.3, the SharePoint Framework Yeoman generator supports scaffoldi
+The dependencies for SPFx v1.4.1 frameworks, tools, and the associated versions don't match the same dependency matrix for the latest versions of SharePoint Framework. In these cases, you may need to install specific versions of the tools. We recommend you install Node.js v8.9.4, gulp v3.9.1, and Yeoman v2.0.6.
 
-> [!NOTE]
-> The Gulp team introduced a separate package, [gulp-cli](https://www.npmjs.com/package/gulp-cli), that should be installed globally. It can be used by projects that use either Gulp v3 & Gulp v4.
->
-> Learn more about the gulp-cli here: [gulpjs/gulp/#2324](https://github.com/gulpjs/gulp/issues/2324).
+## Manuals for web part development and deployment with SharePoint Framework
 
-Microsoft recommends using the most recent version of the Yeoman generator for the SharePoint Framework ([@microsoft/generator-sharepoint](https://www.npmjs.com/package/@microsoft/generator-sharepoint)) that supports creating on-premises projects: SPFx v1.10.0.
+Follow these steps for web part development and deployment with SharePoint Framework:
 
-> [!IMPORTANT]
-> The Yeoman generator for the SharePoint Framework, starting with v1.13.0, only supports projects for SharePoint Online. Learn more about this change in the [SharePoint Framework v1.13 release notes](release-1.13.md). However, SPFx 1.4.1 is only supported on Node.js v6 and Node.js v8. Therefore, you need to get the latest version of the Yeoman generator for the SharePoint Framework (v1.10.0) that works on the same version of Node.js (v6 or v8) that SPFx v1.4.1 is supported on. Solution structure is created then with the v1.4.1 version packages as long as you select the environmen target properly in the Yeoman generator flow.
+1. Prepare the environment for SharePoint Framework development
+2. Develop SharePoint Framework web part
+3. Verify SharePoint Framework web part on local SharePoint workbench
+4. Deploy SharePoint Framework solution to the SharePoint Server
+
+### Prepare the environment for SharePoint Framework development
+
+1. Install Node.js
+
+   Install `Node.js v8.9.4`; if you have `nvm` installed, use `nvm` to install `8.9.4`. Verify the version if you have `Node.js` installed.
+2. Install Yeoman and Gulp
+   
+   Specify these versions:
+   - `npm install -g gulp@3.9.1` 
+   - `npm install -g yo@2.0.6` 
+3. Install Yeoman SharePoint Generator
+   
+   `npm install3 -g @microsoft/generator-sharepoint@1.9.1`. Although not the only option, version `1.9.1` can function appropriately.
+
+### Develop SharePoint Framework web part
+
+1. Create a directory for SharePoint Framework solution with `md spfx-webpart-onprem`.
+
+2. Navigate to the above created directory with `cd spfx-webpart-onprem`.   

```

Comment:
Do not use explicit numbers for bullets, use `1.` for all and let the engine render correctly.